src/main/java/com/github/danielflower/mavenplugins/release/BaseMojo.java

```diff
@@ -162,6 +162,13 @@
     @Parameter(property = "arguments")
     public String arguments;
 
+    /**
+     * <p>Determines the separator between artifactId and groupId<p>
```

Comment:
Ok I will try to do that. That's what i will implement :  
if my custom tag name parameter contains at least substrings "{artifactId}" and "{version}" (minimum required to guarantee uniqueness of a git tag) then do a
```java
tagName=customTagName
        		.replace("{artifactId}", project.getArtifactId())
        		.replace("{groupId}", project.getArtifactId())
        		.replace("{version}", version.releaseVersion());
```
else log a warn and use default value `{artifactid}-{version}`

Reply:
Oh wait, would that complicate the tag lookups in the part of the code that guesses the next build number? (It's been a while since I worked on this so can't really remember)